certified-connectors/PowerToolsPRO/readme.md

```diff
@@ -0,0 +1,30 @@
+## Apptigent PowerTools PRO
+
+Create user-driven solutions that solve everyday business problems in a fraction of the time with the ultimate FREE power user toolkit. Manipulate text, modify collections, format dates and times, convert currency, perform advanced mathematical calculations, shorten URL's, encode strings, and more in just a few clicks. No need to learn complex formulas or expression syntax, PowerTools takes care of all the heavy lifting so users can be instantly more productive. With a powerful toolkit of advanced capabilities at their disposal, every user in the organization will have the ability to deliver outstanding results.
```

Comment:
Does this section's wording need to be changed at all given it's the PRO edition and not free like the LITE edition?

Reply:
My apologies, copy and paste error. 

Corrected in latest commit, do I need to submit a new PR?